app/models/knight.rb

```diff
@@ -1,5 +1,16 @@
 class Knight < Piece
-  # This piece should have some limits on its behavior:
-  # - can only move in an L-shape
-  # - can jump over people (not obstructable)
+  
+  def valid_move?(end_coord)
+
+    end_x_pos = end_coord[0]
+    end_y_pos = end_coord[1]
+    #Check valid coordinates
+    return false unless game.valid_coordinate?(end_coord)
+    #Check move is not a stright line
+    return false if valid_move_direction?(end_coord)
```

Comment:
yeah, because the only other factor is making sure the move is by three spaces, I wanted to make sure it couldn't be three spaces in a straight line.